qtaf-core/src/main/java/de/qytera/qtaf/testng/event_subscriber/TestNGLoggingSubscriber.java

```diff
@@ -27,11 +26,6 @@ public class TestNGLoggingSubscriber implements IEventSubscriber {
      */
     private static final TestSuiteLogCollection testSuiteLogCollection = QtafFactory.getTestSuiteLogCollection();
 
-    /**
```

Comment:
I had to remove this, otherwise it was impossible to mock the logger for message validation (would always be initialized before Mockito could prevent it).